src/devices/command.js

```diff
@@ -26,14 +29,9 @@ export default function buildCommand() {
     return p;
   }
 
-  let broadcastCallback;
-  function registerBroadcastCallback(cb) {
-    broadcastCallback = cb;
-  }
-
   function sendBroadcast(kBodyString) {
-    if (!broadcastCallback) {
-      throw new Error(`sendBroadcast before registerBroadcastCallback`);
+    if (inboundCallback) {
```

Comment:
is this test supposed to be elsewhere? maybe `registerInboundCallback`?

Reply:
Yep. It was a copy-paste error. Corrected.